reservationWeekday.js

```diff
@@ -0,0 +1,18 @@
+const { DateTime } = require("luxon");
+let halls = require("./halls");
+let http = require('http');
+let reservedHalls = [];
+
+for (let i = 0; i < halls.length; i++) {
```

Comment:
Kod for petlje treba imati na umu da je length uvijek za 1 veći od zadnjeg indexa jer počinju s 0.
Zato treba koristiti length -1. 
S druge strane, za ovakve stvari bolje je koristiti array.filter jer ima manje mjesta za greške i pregledniji je